packages/core/src/lib/content/content.engine.ts

```diff
@@ -1,8 +1,12 @@
 import * as Handlebars from 'handlebars';
+import { IAttachmentOptions } from '../..';
```

Comment:
@davidsoderberg Not sure if this import is ok 😅

Reply:
Woops, auto import messed up 😞

Reply:
Happens to me all the time 😅